src/views/LandingPage.vue

```diff
@@ -11,100 +11,75 @@
-            >Vue.js<i class="fab fa-vuejs"></i
-          ></a>
+          <a target="_blank" class="social-icon" href="https://v3.vuejs.org/guide/introduction.html">
+            Vue.js
+            <i class="fab fa-vuejs"></i>
+          </a>
         </div>
         <p class="lead mb-3">
-          The client side runs on Vue.js 3. It utilizes Vuex for state
-          management and Vue Router for navigation. The usage of Vue is a
-          reflection of my recently discovered love of the framework itself.
-          Having mainly been a back end engineer since I began, I needed an
-          approachable solution for the front end. Vue has given me just that.
-        </p>
-        <p class="lead mb-3">
-          Please click on the navigation items to see contributions to the
-          project.
+          The client side runs on Vue.js 3. It utilizes Vuex for state management and Vue Router for navigation. The
+          usage of Vue is a reflection of my recently discovered love of the framework itself. Having mainly been a back
+          end engineer since I began, I needed an approachable solution for the front end. Vue has given me just that.
         </p>
+        <p class="lead mb-3">Please click on the navigation items to see contributions to the project.</p>
       </div>
       <div class="flex-shrink-0">
-        <span class="text-primary"
-          ><a target="_blank" href="https://github.com/actualize-portfolio/alumni_portfolio_vue"
-            >See project on Github</a
-          ></span
-        >
+        <span class="text-primary">
+          <a target="_blank" href="https://github.com/actualize-portfolio/alumni_portfolio_vue">
+            See project on Github
+          </a>
+        </span>
       </div>
-      <div
-        class="d-flex flex-column flex-md-row justify-content-between mb-5"
-      ></div>
+      <div class="d-flex flex-column flex-md-row justify-content-between mb-5"></div>
       <div class="flex-grow-1">
         <h3 class="mb-0">Back End</h3>
         <div class="subheading mb-3">
           <a target="_blank" href="https://rubyonrails.org/">Ruby on Rails</a>
         </div>
         <p class="lead mb-3">
-          The server side is Rails version 6 running in API mode with a Postgres
-          database and a Sidekiq sidecar for asynchronous tasks. It uses a
-          service object interactor pattern and employs some basic
-          authentication with Bcrypt. It is hosted on Heroku.com at
-          <span
-            ><strong>https://alumni-portfolio-api.herokuapp.com/</strong></span
-          >
+          The server side is Rails version 7 running in API mode with a Postgres database and a Sidekiq sidecar for
+          asynchronous tasks. It uses a service object interactor pattern and employs some basic authentication with
+          Bcrypt. It is hosted on Railway.com at
+          <span><strong>https://web-production-ad69e.up.railway.app/api/v1/github_repos</strong></span>
```

Comment:
I know this seems odd because without the path it returns a 404 but, the information is moreso to let anyone curious simply the host for the API. I'm sorry if my direction to try this endpoint was misleading, I was just showing you the endpoint did work.
```suggestion
          <span><strong>https://web-production-ad69e.up.railway.app/api/v1</strong></span>
```